### ISSUE TITLE
What are your thoughts on this article I wrote?

### ISSUE THREAD

**Issue by zaguiini** [kentcdodds/ama#401]
Hey, Kent!

Recently I wrote an article about React application architecture for large scale projects.
Take a look and tell me if you like it, hate it, whatsoever!

Here's the link: https://hackernoon.com/tips-on-react-for-large-scale-projects-3f9ece85983d

**Comment by kentcdodds** [comment]
Hi! I'm afraid I don't have time to give a super solid answer to all my open questions right now, so I just went through and answered all my AMA questions in the form of a video: https://www.youtube.com/watch?v=y23g8L8UtoQ&list=PLV5CVI1eNcJgCrPH_e6d57KRUTiDZgs0u

A few things that folks can do to help make things better for others reading the questions and answers:

1. Link to when I start answering the question. Add `&t=xxmxxs` to the end of the URL like this: https://www.youtube.com/watch?v=y23g8L8UtoQ&list=PLV5CVI1eNcJgCrPH_e6d57KRUTiDZgs0u&t=2m5s
2. Comment with links I referenced
3. Summarize what I talk about

Thanks!
